legend-engine-xt-relationalStore-grammar/src/main/antlr4/org/finos/legend/engine/language/pure/grammar/from/antlr4/connection/datasource/DataSourceSpecificationParserGrammar.g4

```diff
@@ -124,6 +124,22 @@ bigQueryDatasourceSpecification:            BIGQUERY_DSP
                                                     )*
                                                 BRACE_CLOSE
 ;
+spannerDatasourceSpecification:             SPANNER_DATA_SOURCE_SPECIFICATION
+                                                BRACE_OPEN
+                                                    (
```

Comment:
Why do we need host/port for Spanner ? Isn't it just project and instance ?

Reply:
Oh .. this is proxy host/port ? If yes, we should rename to be clear.

Reply:
Also, how do we configure proxy for other databases ?

Reply:
Right now we are using: 
`
dbProxyHost:                                PROXYHOST COLON STRING SEMI_COLON;
dbProxyPort:                                PROXYPORT COLON **STRING** SEMI_COLON;
`
for bigQuery, snowflake
`
dbHost:                                     HOST COLON STRING SEMI_COLON;
dbPort:                                     PORT COLON INTEGER SEMI_COLON;
`
for StaticDataSource, Redshift, and, probabaly Spanner.

The reason why I choosed last possibility it's because on my opinion dbPort should be Integer type instead of string.
Frankly speaking I don't see any real difference between dbHost and dbProxyHost, so I would suggest refactoring deleting one pair or another. Or, maybe, changing proxyPort type from String to Integer. 
Also, I would propose doing this refactoring in terms of **separate** task, because such changes might affect production.

@epsstan what do you think?

Reply:
renamed